MC_Mix.cmd

```diff
@@ -173,37 +173,34 @@ if contains("$scriptlist", "mastercraft") then
 	{
 	action (work) off
 	var temp.room $roomid
-     if ((%water.gone = 1) || (%alcohol.gone = 1) || (%catalyst.gone = 1)) then
-          {
-               if ("$righthandnoun" = "%bowl") then gosub STOW_RIGHT)
-               if ("$lefthandnoun" = "%bowl") then gosub STOW_LEFT)
-          }
+	if ((%water.gone = 1) || (%alcohol.gone = 1) || (%catalyst.gone = 1)) then
+        {
+                if ("$righthandnoun" = "%bowl") then gosub STOW_RIGHT)
+                if ("$lefthandnoun" = "%bowl") then gosub STOW_LEFT)
+        }
 	if %water.gone = 1 then
 	{
-		if !("$righthand" = "Empty" || "$lefthand" = "Empty") then send put my %bowl in my %tool.storage
 		gosub summonwater
-		if (("$righthandnoun" != "%bowl") && ("$lefthandnoun" != "%bowl")) then send get my %bowl from my %tool.storage
+		if "$righthandnoun" != "%bowl" && "$lefthandnoun" != "%bowl" then gosub GET my %bowl from my %tool.storage

```

Comment:
without the gosub get, gets stuck on ...wait or can only type ahead x lines during lag.